Makefile

```diff
@@ -2,16 +2,8 @@ all: dependencies templates
 
 # The release version of `dd-sdk-swift-testing` to use for tests instrumentation.
 DD_SDK_SWIFT_TESTING_VERSION = 2.3.2
```

Comment:
This can be removed as well if we no longer fetch this dependency manually.

Reply:
true.